jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -5819,5 +5869,14 @@ public static Class<?> forPrimitiveName(String typeName) {
 		default -> null;
 		};
 	}
+
+	Method findMethod(boolean isPublic, String methodName, Class<?>... parameterTypes) {
+		try {
+			return getMethodHelper(true, false, null, isPublic ? SearchMethodType.PUBLIC : SearchMethodType.NOTPUBLIC,
```

Comment:
I don't think this is the right interpretation of `isPublic` (which is call `publicOnly` in openjdk). I think `true` means only answer a public method, `false` means answer any method (public or not).

Reply:
> I think true means only answer a public method, false means answer any method (public or not).

Correct, this is what was implemented.
https://github.com/eclipse-openj9/openj9/blob/834b623e9a2b270d958bb1a9d191030341f4144b/jcl/src/java.base/share/classes/java/lang/Class.java#L258-L268

Reply:
More concretely, I think it means the 4th argument should be
```java
    isPublic ? SearchMethodType.PUBLIC : SearchMethodType.ALL
```
which means `SearchMethodType.NOTPUBLIC` would not be used anywhere and `SearchMethodType` could be replaced by `boolean`.

Reply:
Created
* https://github.com/eclipse-openj9/openj9/pull/18569